lang/dmd/files/patch-posix.mak.diff

```diff
@@ -0,0 +1,15 @@
+--- src/posix.mak.orig	2018-05-01 06:42:29.000000000 -0700
++++ src/posix.mak	2018-05-12 12:03:12.000000000 -0700
+@@ -95,9 +95,9 @@
+ G = $(GENERATED)/$(OS)/$(BUILD)/$(MODEL)
+ $(shell mkdir -p $G)
+ 
+-ifeq (osx,$(OS))
+-    export MACOSX_DEPLOYMENT_TARGET=10.9
+-endif
++#ifeq (osx,$(OS))
++#    export MACOSX_DEPLOYMENT_TARGET=10.9
++#endif
```

Comment:
There's really no value to keeping commented-out lines in a patchfile. It makes the patchfile longer (larger) than necessary.

Reply:
@MarcusCalhoun-Lopez Can you fix this and then merge?

Reply:
The commented-out lines have been removed.